PerfTools/AllocMonitor/scripts/edmModuleAllocMonitorAnalyze.py

```diff
@@ -607,22 +619,23 @@ def jsonVisInfo(self,  data):
                 data.findOpenSlotInModGlobals(index,0).append(container[-1])
 
 class PostSourceTransitionParser(SourceTransitionParser):
-    def __init__(self, payload, moduleCentric):
-        super().__init__(payload)
+    def __init__(self, payload, moduleInfos, sourceInfos, moduleCentric):
+        super().__init__(payload, sourceInfos)
         if self.index == -1:
             self.allocInfo = AllocInfo(payload[2:])
         else:
             self.allocInfo = AllocInfo(payload[3:])
         self._moduleCentric = moduleCentric
+        self._sourceInfo = sourceInfos.get(self.index, {})
     def textSpecial(self):
         return "finished"
     def jsonInfo(self, syncs, temp, data):
         start = temp.findTime("source", self.transition, self.index)
         #we do not know the sync yet so have to wait until the framework transition
         if self.transition in [ Phase.construction, Phase.getNextTransition, Phase.destruction, Phase.openFile]:
-            data.insert( "source" , "PoolSource", start, self.time, self.transition, self.index, (0,) , Activity.process, self.allocInfo)
+            data.insert( "source" , self._sourceInfo._cpptype if self._sourceInfo else "Source", start, self.time, self.transition, self.index, (0,) , Activity.process, self.allocInfo)
```

Comment:
Could we just expect the `self._sourceInfo` to exist?
```suggestion
            data.insert( "source" , self._sourceInfo._cpptype, start, self.time, self.transition, self.index, (0,) , Activity.process, self.allocInfo)
```